lambdas/enums/supported_document_types.py

```diff
@@ -20,3 +26,42 @@ def get_from_field_name(value: str):
         if value in SupportedDocumentTypes.list_names():
             return value
         return None
+
+    def get_dynamodb_table_name(self) -> str:
+        """
+        Get the dynamodb table name related to a specific doc_type
+
+        example usage:
+            SupportedDocumentTypes.ARF.get_dynamodb_table_name()
+            (returns "ndr*_DocumentReferenceMetadata")
+
+        result:
+            "ndr*_DocumentReferenceMetadata"
+
+        Eventually we could replace all os.environ["XXX_DYNAMODB_NAME"] calls with this method,
+        so that the logic of resolving table names are controlled in one place.
+        """
+        try:
```

Comment:
For this function do you think it might be sensible to return an array of tablenames (in the case of SupportedDocumentTypes.ALL), this way in DocumentService fetch by doc type we can do a for each on the array of tablenames and query this way now instead of the if statement block that exists currently

Reply:
I quite like this idea! :D  Just implemented it and added a unit test.